pkg/services/accesscontrol/evaluator/evaluator.go

```diff
@@ -30,10 +29,14 @@ func Evaluate(ctx context.Context, ac accesscontrol.AccessControl, user *models.
 	if !ok {
 		return false, nil
 	}
-
 	for _, s := range scope {
 		var match bool
 		for dbScope := range dbScopes {
+			// If there is no scope requires, allow by default
+			if len(dbScope) == 0 {
```

Comment:
This is an explicit check to break the loop when the scope is not defined at all, which by default assumes "don't require scope check"